helm/icees-api/templates/redis-pvc.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
please make this an optional configuration. if the PVC is already created a redeploy will fail.